assets/queries/azureResourceManager/azure_managed_disk_without_encryption/query.rego

```diff
@@ -0,0 +1,37 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Compute/disks"
+	not value.properties.encryptionSettingsCollection.enabled
+
+	issue := prepare_issue(value)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("%s.name=%s%s", [common_lib.concat_path(path), value.name, issue.sk]),
+		"issueType": issue.issueType,
+		"keyExpectedValue": "'encryptionSettingsCollection.enabled' is set to true",
+		"keyActualValue": issue.keyActualValue,
+	}
+}
+
+prepare_issue(resource) = issue {
+	resource.properties.encryptionSettingsCollection.enabled == false
```

Comment:
will it ever reach here in the coverage since we have the `not value.properties.encryptionSettingsCollection.enabled` on line 11?